official/wasm/pkg/goanalysis/goanalysis.go

```diff
@@ -0,0 +1,246 @@
+package goanalysis
```

Comment:
IMO this package is too good for not being in the `github.com/ignite/cli/ignite/pkg`. We can use these functions in more apps.